android/app/src/main/java/im/status/ethereum/MainActivity.java

```diff
@@ -46,6 +46,29 @@ public class MainActivity extends NavigationActivity
     private boolean keepSplash = true;
     private final int SPLASH_DELAY = 3200;
 
+     /**
+       * Returns the name of the main component registered from JavaScript. This is used to schedule
+       * rendering of the component.
+       */
+      protected String getMainComponentName() {
+        return "StatusIm";
+      }
+
+    /**
+       * Returns the instance of the {@link ReactActivityDelegate}. Here we use a util class {@link
+       * DefaultReactActivityDelegate} which allows you to easily enable Fabric and Concurrent React
+       * (aka React 18) with two boolean flags.
+       */
+    /** MainActivity.java:64: error: incompatible types: MainActivity cannot be converted to ReactActivity this, ^
```

Comment:
If we don't need it why do we keep it commented out?

Reply:
removed.